exporters/metric/cortex/testutil_test.go

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
: )

Reply:
:( just saw this in my editor lol. I'll fix it haha